### DISCUSSION TITLE
Can't seem to get this plugin to work reliably?

### DISCUSSION THREAD

**Discussion by tmillr** [pwntester/octo.nvim#576] [General]
I'm gonna put this here for now because I don't have enough concrete details at this point to open an issue.

## Open/Create a New PR

Idk why. I've had this plugin installed for at least a year now, but I hardly ever use it because when I do (90% of the time it's to create a new pr from a fork to the origin/upstream repo), I always get an error? Maybe it's because I'm trying to open/create the pr in the upstream repo (while the current repo is a fork)? Nowadays, the error appears to be originating from the CLI/API (but it could be due to it receiving invalid args or parameters).

## Telescope Pickers/Integration

Also, when I first installed this plugin, I could easily view all of the commands/pickers in telescope, but now I don't see any of them? There are no `Octo` entries now. Has something changed that could be causing this? Or maybe it's just my own setup/config. These days I can only access `Octo` with `:Octo`?

---

Any help would be greatly appreciated. Perhaps I just need to invest more time into debugging or getting this plugin setup and configured properly.

**Comment by williambdean** [comment]
Can you share how you setup up the plugin? And also the error messages that you are seeing
